core/mobile-menu.ts

```diff
@@ -0,0 +1,47 @@
+import { atom, computed } from 'nanostores'
+
+import { fastCategories } from './fast.ts'
+import { router } from './router.ts'
+import { slowCategories } from './slow.ts'
+
+const isMenuShowed = atom<boolean>(false)
```

Comment:
```suggestion
let isMenuShowed = atom<boolean>(false)
```

Reply:
Could you explain me why using `let` better then `const`?

Reply:
Unification. You don’t need to switch between `const` and `let` if you need to change variable later.

Also, I came from Ruby where there is no such separation and use to have only one variable type.